plugins/modules/docker_compose_v2.py

```diff
@@ -465,6 +476,10 @@ def __init__(self, client):
         self.scale = parameters['scale'] or {}
         self.wait = parameters['wait']
         self.wait_timeout = parameters['wait_timeout']
+        if self.compose_version >= LooseVersion('2.32.0'):
+          self.yes = parameters['assume_yes']
+        else:
+          self.yes = False
```

Comment:
I would fail if `assume_yes=true` is supplied with a too old version, instead of silently ignoring it:
```suggestion
        self.yes = parameters['assume_yes']
        if self.compose_version >= LooseVersion('2.32.0') and self.yes:
            self.fail('assume_yes=true needs Docker Compose 2.32.0 or newer, not version %s' % (self.compose_version, ))
```